src/main/kotlin/fi/metatavu/lipsanen/rest/AbstractApi.kt

```diff
@@ -16,6 +17,18 @@ abstract class AbstractApi {
     @Inject
     private lateinit var jsonWebToken: JsonWebToken
 
+    @Inject
+    lateinit var identity: SecurityIdentity
+
+    /**
+     * Checks if user is manager
+     *
+     * @return if user is manager
```

Comment:
```suggestion
     * Checks if user is admin
     *
     * @return if user is admin
```
Manager could be a different role

Reply:
you're right, a typo :)